component/common.libsonnet

```
@@ -156,6 +156,16 @@ local patchPrometheusNetworkPolicy(instanceName) = {
   },
 };
 
+local patchNetworkPolicy(componentName, instanceName, instanceParams) = if std.objectHas(instanceParams[componentName], 'networkPolicy') then
```

```suggestion
local patchNetworkPolicy(componentName, instanceParams) = if std.objectHas(instanceParams[componentName], 'networkPolicy') then
```

`instanceName` isn't used at all